### DISCUSSION TITLE
How to change copilot login in vscode to a new account?

### DISCUSSION THREAD

**Discussion by dcarvalho-bridge-mt** [community/community#57074] [Copilot Conversations]
### Select Topic Area

Question

### Body

I'm facing an issue, I wanna to login with another account in my vscode to use copilot but I could not find a way to do it. Anyone know how?

Invalid copilot token: missing token: 403 
GitHub Copilot could not connect to server. Extension activation failed: "Thank you for using GitHub Copilot. Your subscription has ended. You are currently logged in as...

**Comment by karimdevelops** [comment]


**Reply by dcarvalho-bridge-mt** [reply]
Thank you for you answer. But I'm using a Mac, the only way to do it is to remove all VSCode data?

**Comment by whiterabbitholes** [comment]
I'm sharing what worked for me
In VSCode 2022:
1. open Tools -> Options
2. search 'copilot'
3. set 'Signin with Device Flow' to 'True'
4. now open 'github.com/login/device' (remember to open it on the browser logged in the correct account) and enter the code generated in the VSCode 2022.
5. Done

![image](https://github.com/community/community/assets/12025215/5549e072-a49b-41e3-b49a-4d56b05f1a4f)

**Reply by kmchmk** [reply]
This is Visual Studio not VSCode.
